attributes/network/cddl/1002_idstore.cddl

```diff
@@ -3,42 +3,42 @@ word = tstr
 idstore-recall-phrase = [2*5 word]
 idstore-address = non-anonymous-identity
 idstore-credid = bstr .size (16..1023)
+idstore-pubkey = bstr .cbor COSE_Key
 ; end::types[]
 
 ; tag::store[]
 idstore.store@param = {
-    ; The address associated to the recall phrase
+    ; The address associated with the credential ID and public key
     0 => idstore-address
 
-    ; The credential ID associated to the recall phrase
+    ; The credential ID associated with the recall phrase
     1 => idstore-credid
+
+    ; The public key associated with the recall phrase
+    2 => idstore-pubkey
 }
-idstore.store@return = {
-    ; Recall phrase associated to the address and credential ID
-    0 => idstore-recall-phrase,
-}
+idstore.store@return = idstore-recall-phrase ; The recall phrase associated with the credential ID and public key
 ; end::store[]
 
 ; tag::getFromRecallPhrase[]
-idstore.getFromRecallPhrase@param = {
-    ; The recall phrase to be associated to the credential ID
-    0 => idstore-recall-phrase,
-}
+idstore.getFromRecallPhrase@param = idstore-recall-phrase ; The recall phrase associated with the credential ID and public key
 idstore.idstore.getFromRecallPhrase@return = {
-    ; The credential ID
+    ; The credential ID associated with the recall phrase
     0 => idstore-credid
+
+    ; The public key associated with the recall phrase
+    1 => idstore-pubkey
 }
 ; end::getFromRecallPhrase[]
 
 ; tag::getFromAddress[]
-idstore.getFromAddress@param = {
-    ; The public address associated to the credential ID
-    0 => idstore-address
-}
-
+idstore.getFromAddress@param = idstore-address ; The public address associated with the credential ID and public key
```

Comment:
I prefer to have maps for params and return values because it's better practice to have them everywhere. If you ever need something added you're screwed with backward compatibility.